crates/vm/levm/src/opcode_handlers/arithmetic.rs

```diff
@@ -260,6 +260,27 @@ impl<'a> VM<'a> {
             Ok(OpcodeResult::Continue { pc_increment: 1 })
         }
     }
+
+    pub fn op_clz(&mut self) -> Result<OpcodeResult, VMError> {
+        if self.env.config.fork < Fork::Osaka {
+            return Err(ExceptionalHalt::InvalidOpcode.into());
+        }
+
+        let current_call_frame = &mut self.current_call_frame;
+        current_call_frame.increase_consumed_gas(gas_cost::CLZ)?;
+
+        let [value] = *current_call_frame.stack.pop()?;
+
+        if value != U256::from(0) {
+            current_call_frame
+                .stack
+                .push1(U256::from(value.leading_zeros()))?;
+        } else {
+            current_call_frame.stack.push1(U256::from(256))?;
```

Comment:
do we really need the if here? doesn't value.leading_zeros() return 256 if the value is 0?